src/main/java/com/google/sps/servlets/FreeTimeServlet.java

```diff
@@ -103,13 +103,19 @@ public void doGet(
     FreeTimeUtility freeTimeUtility =
         new FreeTimeUtility(
             timeMin, personalBeginHour, workBeginHour, workEndHour, personalEndHour, numDays);
+    long preAssignedTime = 0;
+    String eventSummary = "Read emails";
```

Comment:
How can you make sure the event summary that you are looking for is consistent with the event summary that you use when creating new events?

Reply:
The summary for the events we are creating are the same as this one. For now this is the check we are using. We assume that the user will not create events with the same summary if they are not related to reading emails. I will add a comment before the loop.